Source/Urho3D/UI/UI.cpp

```diff
@@ -95,7 +95,7 @@ UI::UI(Context* context) :
     dragBeginDistance_(DEFAULT_DRAGBEGIN_DISTANCE),
     mouseButtons_(0),
     lastMouseButtons_(0),
-	maxDoubleClickDist_(FLT_MAX),	
+    maxDoubleClickDist_(99999.0f),	

```

Comment:
There is `M_LARGE_VALUE` constant. Is it acceptable?

Reply:
Awesome! - did not realize that was there - just updated